packages/dynamoose/lib/Table/utilities.ts

```diff
@@ -75,14 +75,29 @@ export async function createTableRequest (table: Table): Promise<DynamoDB.Create
 	}
 
 	// Add stream specification if enabled
-	if (table.getInternalProperties(internalProperties).options.streamOptions.enabled) {
+	if (table.getInternalProperties(internalProperties).options.streamOptions?.enabled) {
 		const streamOptions = table.getInternalProperties(internalProperties).options.streamOptions;
 		object.StreamSpecification = {
 			"StreamEnabled": true,
 			"StreamViewType": streamOptions.type
 		};
 	}
 
+	// Add replication specification if provided
+	if (table.getInternalProperties(internalProperties).options.replication?.regions?.length > 0) {
+		// Ensure streams are enabled for replication
```

Comment:
We just want to throw an error if stream isn't enabled. We don't want to enable it on the users behalf.